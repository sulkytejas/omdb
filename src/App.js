import React, { Component } from 'react';

import Posts from './Components/Posts/';
import Header from './Components/Header/';

class App extends Component {

  render() {
    return (
      <div className="App">
        <Header/>
        <Posts/>
      </div>
    );
  }
}

export default App;
