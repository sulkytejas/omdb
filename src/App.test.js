import React from 'react';
import App from './App';
import renderer from 'react-test-renderer';

it('Snapshot correctly added', () => {
  const component = renderer.create(<App/>);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
