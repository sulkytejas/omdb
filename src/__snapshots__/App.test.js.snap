// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot correctly added 1`] = `
<div
  className="App"
>
  <header
    className="MuiPaper-root-10 MuiPaper-elevation4-16 MuiAppBar-root-1 MuiAppBar-positionStatic-5 MuiAppBar-colorPrimary-8"
  >
    <div
      className="MuiToolbar-root-37 MuiToolbar-regular-39 MuiToolbar-gutters-38"
    >
      <h2
        className="MuiTypography-root-41 MuiTypography-h5-57 MuiTypography-gutterBottom-68"
        style={
          Object {
            "color": "#ffffff",
            "margin": "auto",
            "paddingTop": 10,
          }
        }
      >
        OMDB
      </h2>
    </div>
  </header>
  <div
    style={
      Object {
        "display": "flex",
        "flexDirection": "column",
      }
    }
  >
    <div
      style={
        Object {
          "alignSelf": "center",
          "margin": 20,
        }
      }
    >
      <button
        className="MuiButtonBase-root-102 MuiButton-root-76 MuiButton-outlined-84 MuiButton-outlinedPrimary-85"
        disabled={false}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        style={
          Object {
            "margin": 5,
          }
        }
        tabIndex="0"
        type="button"
      >
        <span
          className="MuiButton-label-77"
        >
          movie
        </span>
        <span
          className="MuiTouchRipple-root-105"
        />
      </button>
      <button
        className="MuiButtonBase-root-102 MuiButton-root-76 MuiButton-outlined-84 MuiButton-outlinedPrimary-85"
        disabled={false}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        style={
          Object {
            "margin": 5,
          }
        }
        tabIndex="0"
        type="button"
      >
        <span
          className="MuiButton-label-77"
        >
          game
        </span>
        <span
          className="MuiTouchRipple-root-105"
        />
      </button>
      <button
        className="MuiButtonBase-root-102 MuiButton-root-76 MuiButton-outlined-84 MuiButton-outlinedPrimary-85"
        disabled={false}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        style={
          Object {
            "margin": 5,
          }
        }
        tabIndex="0"
        type="button"
      >
        <span
          className="MuiButton-label-77"
        >
          series
        </span>
        <span
          className="MuiTouchRipple-root-105"
        />
      </button>
    </div>
  </div>
</div>
`;
